coalib/bears/requirements/PythonRequirement.py

```diff
@@ -0,0 +1,33 @@
+from coalib.bears.PackageRequirement import PackageRequirement
+
+
+class PythonRequirement(PackageRequirement):
+
+    def __init__(self, package, version=""):
+        """
+        :param package: A string with the name of the package to be installed.
+        :param version: A number that contains the version. Leave empty to
+                        install latest.
+        """
+        PackageRequirement.__init__(self, 'pip', package, version)
+
+    @classmethod
+    def multiple(cls, *args):
+        """
+        :param args: Either strings if the latest version is wanted, or tuples
+                     of strings, where the strings consist of the package and
+                     the version number (which is also a string).
```

Comment:
okay this is confusing. I would put some kind of example, like `("package_name", "version")` and remove the part with `where the string consist of the package and ...`